client/src/App.tsx

```diff
@@ -34,6 +35,14 @@ const App: FC = () => {
                 </div>
               }
             />
+          <Route
+              path="/account"
+              element={
+                <div className="Desktop1">
+                  <Desktop1 {...desktop1data} />
```

Comment:
I'm getting an error on this line:
`Cannot find name 'desktop1data'. Did you mean 'Desktop1'?ts(2552)`